<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportScreen</class>
 <widget class="QDialog" name="ReportScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="reportTable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>Nro do Brinco</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Operação</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giro de Gado</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalSpending">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total de gastos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValueTotalEarnings">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R$ 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAverageSpending">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Média de gastos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValueProfit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R$ 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAverageEarnings">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Média de ganhos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProfit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lucro:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValueAverageSpending">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R$ 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValueTotalSpending">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>310</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R$ 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalEarnings">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total de ganhos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValueAverageEarnings">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R$ 0</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateIntervalB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>111</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntervalB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Início do intervalo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntervalE">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>89</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fim do intervalo</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateIntervalE">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
